RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Cumulative Release Notes for rosette-common-java
 
+## 36.0.1
+
+### ROS-217: correct TextDomain to map null inputs to defaults.
```

Comment:
Does a user need to know more?  What if user code was passing null before,
and expecting the `get` method to return what it used to?  That's a breaking
changed that should be mentioned in release notes.  I checked RES and I 
don't think it suffers from this, but I can't speak for rni/rnt, or any client code,
of course.